Sources/Danger/GitHubDSL.swift

```diff
@@ -179,15 +179,6 @@ public struct GitHubUser: Decodable, Equatable {
 
 /// A GitHub team.
 public struct GitHubTeam: Decodable, Equatable {
-    // MARK: - CodingKeys
-
-    enum CodingKeys: String, CodingKey {
-        case id
-        case name
-    }
-
-    // MARK: - Properties
-
     /// The UUID for the team.
     public let id: Int
```

Comment:
<!--
  0 failure: 
  1 warning:  Variable name sho...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/GitHubDSL.swift;
  Line: 183;
-->

- :warning: Variable name should be between 3 and 40 characters long: 'id' (`identifier_name`)